docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md

```diff
@@ -0,0 +1,158 @@
+  application logs to distinguish it from other logs.
+
+- The audit entries logged to `syslog` must be in the RFC 5424 Syslog protocol, with the message in JSON format. The
+  JSON object must contain the following keys: `edgeHostId`, `contentMsg`, `action`, `actor`, `actorType`, `resourceId`,
+  `resourceName`, `resourceKind`. For more information about the Syslog protocol, refer to
+  [RFC 5424 - The Syslog Protocol](https://datatracker.ietf.org/doc/html/rfc5424).
+
+- This how-to is based on the EdgeForge process. We recommend that you familiarize yourself with
+  [EdgeForge](../../edgeforge-workflow/edgeforge-workflow.md) and the process to build Edge artifacts.
+
+- A physical or virtual Linux machine with _AMD64_ (also known as _x86_64_) processor architecture to build the Edge
+  artifacts. You can issue the following command in the terminal to check your processor architecture.
+
+  ```bash
+  uname -m
+  ```
+
+- Minimum hardware configuration of the Linux machine:
+
+  - 4 CPU
+  - 8 GB memory
+  - 150 GB storage
+
+- [Git](https://git-scm.com/downloads). You can ensure git installation by issuing the `git --version` command.
+
+### Procedure
+
+1. Clone the CanvOS repository and change into the directory.
+
+2. Create a file at the path `overlay/files/etc/rsyslog.d/48-audit.conf`. You can change the filename, but it must be a
+   `conf` file and sort alphanumerically before `49-stylus-audit.conf`. For example, `30-my-log.conf`.
+
+<Tabs>
+
+<TabItem value="Send Application Logs to Palette Log File">
+
+3. Populate the file with the following configuration. Replace `your-tag` with the tag you configured for the logs of
+   your own application. If you changed the facility or the severity level from the suggested values, replace them with
+   your actual values.
+
+   ```conf
+   $PrivDropToUser root
+   $PrivDropToGroup root
+   $Umask 0000
+   $template ForwardFormat,"<%pri%>1 %timestamp:::date-rfc3339% %HOSTNAME% %syslogtag% %procid% - - %msg%\n"
+   if ($syslogfacility-text == 'local7' and $syslogseverity-text == 'notice' and $syslogtag contains 'your-tag') then {
+        action(
+            type="omfile"
+            file="/var/log/stylus-audit.log"
+            FileCreateMode="0600"
+            fileowner="root"
+            template="ForwardFormat"
+        )
+    }
+   ```
+
+   Ensure your logs follow the RFC 5424 Syslog protocol, with the message in JSON format. The JSON object must contain
+   the following keys: `edgeHostId`, `contentMsg`, `action`, `actor`, `actorType`, `resourceId`, `resourceName`,
+   `resourceKind`. If your log entries do not follow this RFE 5424 Syslog protocol, or do not contain the necessary
+   keys, they will still be logged, but will not show up in Local UI.
+
+</TabItem>
+
+<TabItem value="Send Palette Logs to Another File">
+
+3. Populate the file with the following configuration. Replace `your-file` with the name of your log file.
```

Comment:
@lennessyy , this should be point 4, 3 is already defined. Also point 2,3 and 4 are optional

Reply:
Hi Arun, so these are two tabs I made for two different use cases. One is to send stylus logs to another file, and the other is to send app logs to the stylus log file. User can do either or both. I just didn't want to create two tasks that are basically identically save for the conf file.

Can you explain what you mean by point 2, 3 ,4 are optional? That leaves step 1 which is just to clone the repo.